### PR TITLE
ReincubateLtd.BlackBerryBackupExtractor version 2.0.7.3

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98711)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The package installs normally, but requires `Microsoft.DotNet.Framework.DeveloperPack_4` to run:
image

Pending:
* https://github.com/microsoft/winget-cli/issues/163

**Comment by upintheairsheep** [comment]
Ok

**Comment by stephengillie** [comment]
> 12/20/2023 5:22:03 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   F1CE6D3956C9EC05C7FDC5CC58828B62E698D9A9B27733B2DF03166F9242F2A3
Expected hash: 90040c79122a6e9261c796210b35aeb7a488989647d9b5f363688f6aaf36e9ac

File: manifests/r/ReincubateLtd/BlackBerryBackupExtractor/2.0.7.3/ReincubateLtd.BlackBerryBackupExtractor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ReincubateLtd.BlackBerryBackupExtractor
+PackageVersion: 2.0.7.3
+Installers:
+- InstallerUrl: https://reincubate.com/res/labs/bbbe/bbbe-latest.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: 90040C79122A6E9261C796210B35AEB7A488989647D9B5F363688F6AAF36E9AC
```
Comment (stephengillie) [inline manifests/r/ReincubateLtd/BlackBerryBackupExtractor/2.0.7.3/ReincubateLtd.BlackBerryBackupExtractor.installer.yaml:10]:
```suggestion
  InstallerSha256: F1CE6D3956C9EC05C7FDC5CC58828B62E698D9A9B27733B2DF03166F9242F2A3
```

**Pull request closed without merging**